cohortextractor/backends/databricks.py

```diff
@@ -0,0 +1,66 @@
+from cohortextractor.backends.base import BaseBackend, Column, MappedTable, QueryTable
+from cohortextractor.query_engines.mssql import MssqlQueryEngine
+
+
+class DatabricksBackend(BaseBackend):
+    backend_id = "databricks"
+    # This will have to change, obviously, but it will do to start with
+    query_engine_class = MssqlQueryEngine
+    patient_join_column = "patient_id"
+
+    patients = QueryTable(
+        columns=dict(
+            date_of_birth=Column("date"),
+        ),
+        query="""
+        SELECT
+            Person_ID AS patient_id, MAX(PatientDoB) AS date_of_birth
+        FROM
+            PCAREMEDS_pcaremeds
+        GROUP BY
+            Person_ID
+        """,
+    )
+
+    prescriptions = MappedTable(
+        source="PCAREMEDS_pcaremeds",
+        columns=dict(
+            patient_id=Column("integer", source="Person_ID"),
+            prescribed_dmd_code=Column(
+                "code", source="PrescribeddmdCode", system="dmd"
+            ),
+            processing_date=Column("date", source="ProcessingPeriodDate"),
+        ),
+    )
+
+    hospital_admissions = QueryTable(
+        columns=dict(
+            admission_date=Column("date"),
+            primary_diagnosis=Column("code", system="icd10"),
+            admission_method=Column("integer"),
+            episode_is_finished=Column("boolean"),
+            spell_id=Column("integer"),
+        ),
+        query="\nUNION ALL\n".join(
+            f"""
+            SELECT
+                mps.PERSON_ID AS patient_id,
+                apc.ADMIDATE AS admission_date,
+                apc.DIAG_4_01 AS primary_diagnosis,
+                apc.ADMINMETH AS admission_method,
+                apc.FAE AS episode_is_finished,
+                apc_otr.SUSSPELLID AS spell_id
+            FROM
+                HES_AHAS_hes_apc_{year} AS apc
+            JOIN
+                HES_AHAS_MPS_hes_apc_{year} AS mps
+            ON
+                apc.EPIKEY = mps.EPIKEY
+            LEFT JOIN
+                HES_AHAS_hes_apc_otr_{year} AS apc_otr
+            ON
+                apc.EPIKEY = apc_otr.EPIKEY
+            """
+            for year in ["1920"]
```

Comment:
Only one year? Why the `UNION`?

Reply:
Sorry yes, just updated with more comments.